app/config/locale/translations/tel.countries.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+return [
+    'AF' => 'ఆఫ్ఘనిస్తాన్',
+    'AO' => 'అంగోలా',
+    'AL' => 'అల్బేనియా',
+    'AD' => 'అండోరా',
+    'AE' => 'యునైటెడ్ అరబ్ ఎమిరేట్స్',
+    'AR' => 'అర్జెంటీనా',
+    'AM' => 'అర్మేనియా',
+    'AG' => 'ఆంటిగ్వా మరియు బార్బుడా',
+    'AU' => 'ఆస్ట్రేలియా',
+    'AT' => 'ఆస్ట్రియా',
+    'AZ' => 'అజర్‌బైజాన్',
+    'BI' => 'బురుండి',
+    'BE' => 'బెల్జియం',
+    'BJ' => 'బాలురు',
+    'BF' => 'బుర్కినా ఫాసో',
+    'BD' => 'బంగ్లాదేశ్',
+    'BG' => 'బల్గేరియా',
+    'BH' => 'రెండు సముద్రాలు',
+    'BS' => 'బహామాస్',
+    'BA' => 'బోస్నియా మరియు హెర్జెగోవినా',
+    'BY' => 'బెలారస్',
+    'BZ' => 'దయచేసి',
+    'BO' => 'బొలీవియా',
+    'BR' => 'బ్రెజిల్',
+    'BB' => 'బార్బడోస్',
+    'BN' => 'బ్రూనై',
+    'BT' => 'భూటాన్',
+    'BW' => 'బోట్స్వానా',
+    'CF' => 'సెంట్రల్ ఆఫ్రికన్ రిపబ్లిక్',
+    'CA' => 'కెనడా',
+    'CH' => 'స్విట్జర్లాండ్',
+    'CL' => 'చిలీ',
+    'CN' => 'చైనా',
+    'CI' => 'కోట్ డి ఐవోరీ',
+    'CM' => 'కామెరూన్',
+    'CD' => 'కాంగో డెమొక్రాటిక్ రిపబ్లిక్',
+    'CG' => 'కాంగో రిపబ్లిక్',
+    'CO' => 'కొలంబియా',
+    'KM' => 'కొమొరోస్',
+    'CV' => 'ఆకుపచ్చ తల',
+    'CR' => 'కోస్టా రికా',
+    'CU' => 'క్యూబా',
+    'CY' => 'సైప్రస్',
+    'CZ' => 'చెకియా',
+    'DE' => 'జర్మనీ',
+    'DJ' => 'జిబౌటి',
+    'DM' => 'డొమినికా',
+    'DK' => 'డెన్మార్క్',
+    'DO' => 'డొమినికన్ రిపబ్లిక్',
+    'DZ' => 'అల్జీరియా',
+    'EC' => 'ఈక్వెడార్',
+    'EG' => 'ఈజిప్ట్',
+    'ER' => 'ఎరిట్రియా',
+    'ES' => 'స్పెయిన్',
+    'EE' => 'ఎస్టోనియా',
+    'ET' => 'ఇథియోపియా',
+    'FI' => 'ఫిన్లాండ్',
+    'FJ' => 'ఫిజీ',
+    'FR' => 'ఫ్రాన్స్',
+    'FM' => 'మైక్రోనేషియా',
+    'GA' => 'గాబన్',
+    'GB' => 'యునైటెడ్ రాజ్యం',
+    'GE' => 'జార్జియా',
+    'GH' => 'ఘనా',
+    'GN' => 'గినియా',
+    'GM' => 'గాంబియా',
+    'GW' => 'గినియా-బిసావు',
+    'GQ' => 'ఈక్వటోరియల్ గినియా',
+    'GR' => 'గ్రీస్',
+    'GD' => 'గ్రెనడా',
+    'GT' => 'గ్వాటెమాల',
+    'GY' => 'గయానా',
+    'HN' => 'హోండురాస్',
+    'HR' => 'క్రొయేషియా',
+    'HT' => 'హైతీ',
+    'HU' => 'హంగరీ',
+    'ID' => 'ఇండోనేషియా',
+    'IN' => 'భారతదేశం',
+    'IE' => 'ఐర్లాండ్',
+    'IR' => 'ఇరాన్',
+    'IQ' => 'ఇరాక్',
+    'IS' => 'ఐస్లాండ్',
+    'IL' => 'ఇజ్రాయెల్',
+    'IT' => 'ఇటలీ',
+    'JM' => 'జమైకా',
+    'JO' => 'జోర్డాన్',
+    'JP' => 'జపాన్',
+    'KZ' => 'కజాఖ్స్తాన్',
+    'KE' => 'కెన్యా',
+    'KG' => 'కిర్గిజ్స్తాన్',
+    'KH' => 'కంబోడియా',
+    'KI' => 'కిరిబాటి',
+    'KN' => 'సెయింట్ కిట్స్ మరియు నెవిస్',
+    'KR' => 'దక్షిణ కొరియా',
+    'KW' => 'కువైట్',
+    'LA' => 'లావోస్',
+    'LB' => 'లెబనాన్',
+    'LR' => 'లైబీరియా',
+    'LY' => 'లిబియా',
+    'LC' => 'సెయింట్ లూసియా',
+    'LI' => 'లిచ్టెన్స్టెయిన్',
+    'LK' => 'శ్రీలంక',
+    'LS' => 'లెసోతో',
+    'LT' => 'లిథువేనియా',
+    'LU' => 'లక్సెంబర్గ్',
+    'LV' => 'లాట్వియా',
+    'MA' => 'మొరాకో, వెస్ట్, సూర్యాస్తమయం',
```

Comment:
the 'మొరాకో, వెస్ట్, సూర్యాస్తమయం', you should remove the comma from this

Reply:
@spielers I have done suggested changes, please look that changes.